src/App.js

```diff
@@ -1,10 +1,17 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
 import LandingPage from "./pages/LandingPage";
+import VideoListingPage from "./pages/VideoListingPage";
+import VideoPage from "./pages/VideoPage";
 
 function App() {
   return (
     <div className="App">
-      <LandingPage />
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="/videoListingPage" element={<VideoListingPage />} />
+        <Route path="/videoPage" element={<VideoPage />} />
+      </Routes>
```

Comment:
Instead of adding routes in the App.js file, you could move it to a separate file say AppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.